### DISCUSSION TITLE
ORB SLAM Version

### DISCUSSION THREAD

**Discussion by aravindSolteq** [stella-cv/stella_vslam#184] [Q&A] [answered]
Hi the framework input is orb_vocab.fbow, Which algorithm version does this follow , ORB SLAM2 or ORB SLAM3?

**Comment by ymd-stella** [comment, marked as answer]
See https://github.com/xdspacelab/openvslam/issues/121.
The author says, "I did it with the KITTI sequences".
I haven't checked, but I think it is different from the vocabulary used in ORB_SLAM2 and ORB_SLAM3.
